frontend/packages/kubevirt-plugin/integration-tests/tests/vm.non-admin.scenario.ts

```diff
@@ -0,0 +1,82 @@
+import { execSync } from 'child_process';
+import { browser, ExpectedConditions as until } from 'protractor';
+import { appHost, testName } from '@console/internal-integration-tests/protractor.conf';
+import * as loginView from '@console/internal-integration-tests/views/login.view';
+import { withResource, removeLeakedResources } from '@console/shared/src/test-utils/utils';
+import { isLoaded } from '@console/internal-integration-tests/views/crud.view';
+import { isRestrictedAccess } from '../views/vms.list.view';
+import { createProject } from './utils/utils';
+import { vmConfig, getProvisionConfigs } from './vm.wizard.configs';
+import { ProvisionConfigName } from './utils/constants/wizard';
+import { VirtualMachine } from './models/virtualMachine';
+import { VM_ACTION, VM_STATUS, JASMINE_EXTENDED_TIMEOUT_INTERVAL } from './utils/consts';
+
+const testNonAdminNamespace = `${testName}-non-admin`;
+const {
+  BRIDGE_HTPASSWD_IDP = 'test',
+  BRIDGE_HTPASSWD_USERNAME = 'test',
+  BRIDGE_HTPASSWD_PASSWORD = 'test',
+} = process.env;
+
+describe('Kubevirt non-admin Flow', () => {
+  const leakedResources = new Set<string>();
+  const configName = ProvisionConfigName.URL;
+  const provisionConfigs = getProvisionConfigs(testName);
+  const provisionConfig = provisionConfigs.get(configName);
+  provisionConfig.networkResources = [];
+  provisionConfig.storageResources = [];
+
+  const vm1Config = vmConfig(configName.toLowerCase(), provisionConfig, testNonAdminNamespace);
```

Comment:
I changed the order of parameters here to match other methods:
`vmConfig(configName.toLowerCase(), testNonAdminNamespace, provisionConfig)`